creator-node/src/logging.js

```diff
@@ -3,13 +3,23 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+function StdOutWithLevelName() {}
+StdOutWithLevelName.prototype.write = function (data) {
+  const logObject = JSON.parse(data)
```

Comment:
could we use type: raw instead of stream? https://github.com/trentm/node-bunyan#stream-type-raw. if we can avoid JSON.parse/JSON.stringify that's a huge amount of computation load lifted.

Reply:
if we're forced to use stream with JSON parse/stringify we should try/catch the JSON.parse here [in case it throws](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#exceptions). in the event of an error we should directly write data to process.stdout

Reply:
I've added a second stream handler that manipulates the raw `Object` before it is put through `JSON.stringify`, but my current output is as follows:

> {"0":"{","1":"\"","2":"n","3":"a","4":"m","5":"e","6":"\"","7":":","8":"\"","9":"a","10":"u","11":"d","12":"i","13":"u","14":"s","15":"_","16":"c","17":"r","18":"e","19":"a","20":"t","21":"o","22":"r","23":"_","24":"n","25":"o","26":"d","27":"e","28":"\"","29":",","30":"\"","31":"h","32":"o","33":"s","34":"t","35":"n","36":"a","37":"m","38":"e","39":"\"","40":":","41":"\"","42":"5","43":"c","44":"3","45":"d","46":"d","47":"c","48":"9","49":"b","50":"b","51":"9","52":"4","53":"e","54":"\"","55":",","56":"\"","57":"p","58":"i","59":"d","60":"\"","61":":","62":"5","63":"6","64":",","65":"\"","66":"l","67":"e","68":"v","69":"e","70":"l","71":"\"","72":":","73":"3","74":"0","75":",","76":"\"","77":"m","78":"s","79":"g","80":"\"","81":":","82":"\"","83":"S","84":"n","85":"a","86":"p","87":"b","88":"a","89":"c","90":"k","91":"S","92":"M","93":":","94":" ","95":"p","96":"r","97":"o","98":"c","99":"e","100":"s","101":"s","102":"S","103":"t","104":"a","105":"t","106":"e","107":"M","108":"a","109":"c","110":"h","111":"i","112":"n","113":"e","114":"O","115":"p","116":"e","117":"r","118":"a","119":"t","120":"i","121":"o","122":"n","123":"(","124":")","125":" ","126":"7","127":" ","128":"b","129":"u","130":"i","131":"l","132":"d","133":"R","134":"e","135":"p","136":"l","137":"i","138":"c","139":"a","140":"S","141":"e","142":"t","143":"N","144":"o","145":"d","146":"e","147":"s","148":"T","149":"o","150":"U","151":"s","152":"e","153":"r","154":"W","155":"a","156":"l","157":"l","158":"e","159":"t","160":"s","161":"M","162":"a","163":"p","164":"(","165":")","166":" ","167":"S","168":"u","169":"c","170":"c","171":"e","172":"s","173":"s","174":" ","175":"-","176":" ","177":"D","178":"a","179":"t","180":"a","181":" ","182":"{","183":"\\","184":"\"","185":"n","186":"u","187":"m","188":"R","189":"e","190":"p","191":"l","192":"i","193":"c","194":"a","195":"S","196":"e","197":"t","198":"N","199":"o","200":"d","201":"e","202":"s","203":"\\","204":"\"","205":":","206":"0","207":"}","208":" ","209":"-","210":" ","211":"D","212":"u","213":"r","214":"a","215":"t","216":"i","217":"o","218":"n","219":" ","220":"1","221":"m","222":"s","223":"\"","224":",","225":"\"","226":"t","227":"i","228":"m","229":"e","230":"\"","231":":","232":"\"","233":"2","234":"0","235":"2","236":"2","237":"-","238":"0","239":"5","240":"-","241":"2","242":"3","243":"T","244":"2","245":"1","246":":","247":"2","248":"7","249":":","250":"1","251":"7","252":".","253":"0","254":"0","255":"7","256":"Z","257":"\"","258":",","259":"\"","260":"v","261":"\"","262":":","263":"0","264":"}","265":"\n"}

Let me know if we should:

* return to the previous stream handler,
* pursue this "raw" path further,
* or find a replacement for Bunyan.